app/src/docs/v1.api-spec.yaml

```diff
@@ -404,7 +404,7 @@ paths:
       description: >-
         Creates a copy and new version of the object with the given metadata removed.
         Multiple Key/Value pairs can be provided in the header for the metadata.
-        If no metadata headers are given then all metadata will be removed.
+        If no metadata headers are given then all metadata will be removed except `name` and `id` which always exist.
```

Comment:
If no metadata headers are given then all metadata will be removed.
Metadata headers `name` and `id` are mandatory and will always persist.